augeas-mode.el

```diff
@@ -74,7 +74,7 @@ Argument IGNORE is not used here."
 (defvar augeas-mode-syntax-table
   (let ((st (make-syntax-table)))
       (modify-syntax-entry ?_ "w"    st)
-      (modify-syntax-entry ?\" "_"   st)
+      (modify-syntax-entry ?\" "."   st)
```

Comment:
I tried some more things. Basically I switched the order of the two regexps. Maybe you would like to test the code from my [string-font-lock-fix](https://github.com/smoeding/emacs-augeas-mode/blob/string-font-lock-fix/augeas-mode.el) branch?